addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -55,6 +51,11 @@ def _compute_csv_created(self):
         for record in self:
             record.l10n_id_csv_created = bool(record.l10n_id_attachment_id)
 
+    @api.depends('partner_id')
+    def _compute_kode_transaksi(self):
+        for move in self:
+            move.l10n_id_kode_transaksi = move.partner_id.l10n_id_kode_transaksi
```

Comment:
Testing it, I still have a problem when I am overwriting the value, then it will be set back to the value on the partner_id while it should not